src/components/timeline/TrackMemoryGraph.js

```diff
@@ -269,7 +269,20 @@ class TrackMemoryGraphImpl extends React.PureComponent<Props, State> {
       // We are outside the range of the samples, do not display hover information.
       this.setState({ hoveredCounter: null });
     } else {
-      let hoveredCounter = bisection.right(samples.time, timeAtMouse);
+      let hoveredCounter;
+      const bisectionCounter = bisection.right(samples.time, timeAtMouse);
```

Comment:
Can you please add a comment, that could be something like this:
```js
// When the mouse pointer hovers between 2 points, we want to select the point that's closest.
```

Reply:
Added.